helpers/dateTime.js

```diff
@@ -161,6 +161,44 @@ export function getUTCDateTimeFromLocalDateTime(date, time) {
 	return formatDateTimeInISO(utcDateTime);
 }
 
+export function getUTCDateTimeRange(type, diff) {
+	if (!type || (!diff && diff !== 0)) return {};
+	if (type !== 'hours' && type !== 'days' && type !== 'weeks' && type !== 'months') return {};
+
+	if (type === 'days' && diff === 0) {
+		// assume "today" (midnight in user's time to 23:59:59 in user's time)
+		const today = formatDateInISO(getToday());
+		const startValue = getUTCDateTimeFromLocalDateTime(today, '0:0:0');
+		const endValue = getUTCDateTimeFromLocalDateTime(today, '23:59:59');
+		return { startValue, endValue };
+	}
+
+	/**
+	 * endValue = now in UTC string
+	 * startValue = now minus number of hours/days/weeks/months in diff then converted to UTC string
+	 */
```

Comment:
Please confirm my assumption/logic with this:
If we get `new Date()` in ISO string, that would be the UTC time for now and timezone wouldn't matter (the cases where this usually gets complicated is when the user has a different LMS timezone set from their browser's timezone, which was what we handled quite a bit with date-time inputs). We therefore are safe to just use that as the end value, then get the start value, without having to consider the user's timezone.

Reply:
I _think_ so.